package.json

```diff
@@ -99,12 +99,17 @@
   },
   "devDependencies": {
     "babel-eslint": "^7.2.1",
+    "babel-jest": "^19.0.0",
     "babel-plugin-lodash": "^3.2.11",
-    "babel-preset-stage-2": "^6.22.0",
```

Comment:
This is being used above - it probably still needs to be included or babel will be upset.